{{cookiecutter.extension_name}}/setup.py

```diff
@@ -0,0 +1,86 @@
+"""
+Setup Module to setup Python Handlers for the {{ cookiecutter.extension_name }} extension.
+"""
+import os
+
+from setupbase import (
+    create_cmdclass, install_npm, ensure_targets,
+    combine_commands, ensure_python, get_version,
+)
+import setuptools
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+# The name of the project
+name="{{ cookiecutter.extension_name|replace("-", "_") }}"
+
+# Ensure a valid python version
+ensure_python(">=3.6")
```

Comment:
Should this be `3.6` or `3.5`? (same as jupyterlab: https://github.com/jupyterlab/jupyterlab/blob/04d2bfa738d01e231081a8d5604127465a7aeed3/setup.py#L28)

Reply:
3.5 has another six months left: https://devguide.python.org/#status-of-python-branches

Reply:
Thanks for the tip. I thought it was already unsupported.